pint_xarray/tests/test_accessors.py

```diff
@@ -1516,12 +1521,13 @@ def test_reindex_like(obj, units, other, other_units, expected, expected_units,
             {"x": "dm", "y": "s"},
             {"x": Quantity([1, 3, 5], "m"), "y": Quantity([0, 2], "min")},
             xr.DataArray(
-                [[np.nan, 1], [np.nan, 5], [np.nan, np.nan]],
+                [[0, 1], [0, 5], [0, 0]],
                 dims=("x", "y"),
                 coords={"x": ("x", [1, 3, 5]), "y": ("y", [0, 2])},
             ),
             {"x": "m", "y": "min"},
             None,
+            {"bounds_error": False, "fill_value": 0},
```

Comment:
I wonder whether it would be better to add a separate parameter set, with an explicit name like `DataArray-other parameters`